.gitlab/e2e.yml

```diff
@@ -2,11 +2,18 @@
 # e2e stage
 # Contains jobs which runs e2e tests on our Docker images.
 
+.container_needs:
+  needs:
+    - qa_agent
+    - qa_dca
+    - qa_dogstatsd
+
 .k8s_e2e_template:
   stage: e2e
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/gitlab_agent_deploy:$DATADOG_AGENT_BUILDERS
   tags: ["arch:amd64"]
   dependencies: []
+  extends: .container_needs
```

Comment:
The `k8s_e2e*` jobs do not refer to jobs run with `new-e2e`. So I am not sure these jobs actually use the images pushed on the `qa` environment

Reply:
the jobs extending `k8s_e2e_template` seem to refer to agent-image and dca-image built during these steps, that's why I put this reference